.github/workflows/lint-only.yaml

```
@@ -0,0 +1,48 @@
+---
+
+name: Lint Charts 
+
+on:
+  push:
+    branches: []
```

actionlint (1.7.11)

[error] "branches" section should not be empty

(syntax-check)